streamz/core.py

```diff
@@ -546,8 +447,10 @@ def update(self, x, who=None):
             return []
 
 
+@Stream.register_api
 class map(Stream):
-    def __init__(self, func, child, args=(), **kwargs):
+    """ Apply a function to every element in the stream """
+    def __init__(self, child, func, *args, **kwargs):
```

Comment:
This is an API break. Should we tag it as such somewhere or mention in commit? (same for `filter` etc)

Reply:
If the project was further along, yes.  However at this early stage I plan to move quickly and break things.  I apologize if this causes you all some stress as early adopters.  You may be better served by relying on a previously released version for a while.

I apologize for my lack of rigor here.  I am fairly time constrained.

Reply:
For me it's no problem (worst case I can keep a fork). I just thought I would point it out. I wasn't sure which stage of development `streamz` is in at this point since it seems to work fairly well (and we're using it and have deployed an initial prototype).

Reply:
(and thanks for this great library!)

Reply:
We may want to release the library for a `0.2.0` backdated before this merge? That way we have something before the break.